pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -45,6 +45,8 @@ const (
 	GET_USERS      AgentCommand = "guest-get-users"
 	GET_FILESYSTEM AgentCommand = "guest-get-fsinfo"
 	GET_AGENT      AgentCommand = "guest-info"
+
+	pollInitialInterval = 10 * time.Second
```

Comment:
const?

Reply:
Adding the keyword `pollInitialInterval` will enforce it to be one in case someone
write to it, isnt it ?

Reply:
Sorry, I do not understand what you are asking.

Reply:
`const pollInitialInterval = 10 * time.Second`
Once you add the `const` word, the compiler will protect it if someone assign to it

Reply:
These are 4 comments too much.
As I mentioned in the second comment of this thread, it is already a constant. You just need to look a few lines above to see in what block it is located.

Reply:
yes sorry was afk when i figured that what i might overlooked